Makefile

```diff
@@ -262,7 +262,7 @@ generate: ## Generate code
 .PHONY: generate-go
 generate-go: $(CONTROLLER_GEN)
 generate-go: ## Generate golang sources
-	go generate ./api/...
+	go generate github.com/vmware-tanzu/vm-operator/api/...
```

Comment:
Why this change?

Reply:
So this was addressed just now in #523 . You can undo your change @dougm . Apparently between Go 1.21 and Go 1.22, cross-module commands were restricted. From a Slack conversation with @dilyar85:

> Hi Andrew. I noticed that running `go generate ./api/...` in VMOP returns the following error for Go 1.22 (the previous version 1.21 doesn’t have this issue):
> 
> ```shell
> $ go version
> go version go1.22.3 darwin/amd64
> 
> $ go generate ./api/...
> pattern ./api/...: main module (github.com/vmware-tanzu/vm-operator) does not contain package > github.com/vmware-tanzu/vm-operator/api
> ```
> 
> I checked the [Go 1.22 Release Notes](https://tip.golang.org/doc/go1.22) but didn’t see anything related. Also did some searching online and found no relevant results.
> Since we use Go 1.22 in our GitHub CI, this is failing the verify-codegen workflow, which executes this go generate command when calling the `make generate` target. See https://github.com/vmware-tanzu/vm-operator/actions/runs/9103082015/job/25024152057?pr=522
[9:11](https://vmware.slack.com/archives/D02H2M35LFM/p1715868672333429)
I also tried upgrading both VMOP and API modules to 1.22 (matching the CI version) but it didn’t help either.

Reply:
Nice, done